oci/oci.go

```diff
@@ -42,6 +42,8 @@ const (
 	ContainerExitsDir = "/var/run/crio/exits"
 	// ContainerAttachSocketDir is the location for container attach sockets
 	ContainerAttachSocketDir = "/var/run/crio"
+	// BufSize is the size of buffers passed in to socekts
+	BufSize = 8192
```

Comment:
Do we need to keep this one in go? I think this is only needed in conmon

Reply:
> Do we need to keep this one in go? I think this is only needed in conmon

It's used over [here][1].

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1414/files#diff-702431da3a2e35392cf253955d15f774L43

Reply:
Alright. The buffer size looks good, but the log size connection seems a little nonsensical.